operator/apis/loki/v1/lokistack_types.go

```diff
@@ -1070,12 +1070,12 @@ const (
 	ReasonMissingObjectStorageSecret LokiStackConditionReason = "MissingObjectStorageSecret"
 	// ReasonInvalidObjectStorageSecret when the format of the secret is invalid.
 	ReasonInvalidObjectStorageSecret LokiStackConditionReason = "InvalidObjectStorageSecret"
-	// ReasonMissingCredentialsRequest when the required request for managed auth credentials to object
+	// ReasonMissingCredentialsRequest when the required request for token cco auth credentials to object
 	// storage is missing.
 	ReasonMissingCredentialsRequest LokiStackConditionReason = "MissingCredentialsRequest"
-	// ReasonMissingManagedAuthSecret when the required secret for managed auth credentials to object
+	// ReasonMissingTokenCCOAuthSecret when the required secret for token cco auth credentials to object
 	// storage is missing.
```

Comment:
Making this a bit more detailed as well:

```suggestion
	// ReasonMissingTokenCCOAuthSecret when the secret generated by CCO for token authentication is missing.
	// This is usually a transient error because the secret is not immediately available after creating the
	// CredentialsRequest, but it can persist if the CCO or its configuration is incorrect.
```